EN/02-docker/installing-docker.adoc

```diff
@@ -0,0 +1,85 @@
+Installing Docker
```

Comment:
well, IMO, I think the Installation/Docker(recommended) should point to the Docker section and should be redone.